db/seeds.rb

```diff
@@ -1,13 +1,27 @@
 Campaign.destroy_all
+Genre.destroy_all
 User.destroy_all
 
 fan = User.create(email: 'fan@venue.se', password: 'my-password', role: 'fan')
 artist = User.create(email: 'artist@venue.se', password: 'my-password', role: 'artist')
 admin = User.create(email: 'admin@venue.se', password: 'my-password', role: 'admin')
 
+rock = Genre.create(name: "rock")
+metal = Genre.create(name: "metal")
+punk = Genre.create(name: "punk")
+indie = Genre.create(name: "indie")
+pop = Genre.create(name: "pop")
+electronic = Genre.create(name: "electronic")
+hip hop = Genre.create(name: "hip hop")
+rap = Genre.create(name: "rap")
+classic = Genre.create(name: "classic")
+country = Genre.create(name: "country")
+jazz = Genre.create(name: "jazz")
+blues = Genre.create(name: "blues")
+
 artist.performers.create(
     name: 'Clare Cunningham',
-    genre: 'Rock',
+    genre: rock,
```

Comment:
This sets the value ´rock` as a string on that performer. Is that what you want? Shouldn't this be an association? Especially if we want to be able to search by genre it should be IMO